test/common/modules/private_api_recorder.js

```diff
@@ -82,13 +82,15 @@ define(['test/support/output_directory_paths'], function (outputDirectoryPaths)
     'read.LiveMap._dataRef.data',
     'read.EventEmitter.events',
     'read.Platform.Config.push',
+    'read.ProtocolMessage.channelSerial',
     'read.Realtime._transports',
     'read.auth.authOptions.authUrl',
     'read.auth.key',
     'read.auth.method',
     'read.auth.tokenParams.version',
     'read.channel.channelOptions',
     'read.channel.channelOptions.cipher',
+    'read.channel.properties.channelSerial',
```

Comment:
Could you put a comment next to this one please mentioning that this is meant to be public API, with a link to https://github.com/ably/ably-js/issues/2018?